### DISCUSSION TITLE
移植到华硕tinker borad 2

### DISCUSSION THREAD

**Discussion by mycrl** [immortalwrt/immortalwrt#815] [Q&A]
我最近在做一项尝试，我想让openwrt跑在tinker borad 2上面，我找到了同为rk3399芯片的设备nano pi r4s，我尝试了这个设备的openwrt image，发现无法启动，我对openwrt不了解，请问可能的原因是什么？uboot还是设备树？

或者我需要做些什么？

**Comment by xlighting2017** [comment]
可以参考 Armbian 那边的进展 （用别的板子的 image 即使能启动也只是奇迹

https://forum.armbian.com/topic/17448-is-asus-tinker-board-2-supported-by-armbian/#comment-122107

https://tinker-board.asus.com/forum/index.php?/topic/15196-armbian-for-tinkerboard-2-wip/

给开发者 打钱，提供设备， 都是能使其获得支持的好办法

**Reply by mycrl** [reply]
我使用nanopi r4s的sd openwrt image能在tinker borad 2里面成功启动，但是我得不到任何输出，并且网卡也无法正常工作，但是led指示灯是在正常闪烁，我检查了sd卡上面的文件系统，发现openwrt初始化已经完成，所以我怀疑是否跟驱动相关，在这种情况下有没有可能从现有的tinker os（官方image）里面移植驱动过来？

**Reply by xlighting2017** [reply]
如果你说的移植是 复制/粘贴 文件的话， 还是 给开发者打钱/提供设备吧
